.github/workflows/reports-scheduler-test-and-build-workflow.yml

```diff
@@ -7,13 +7,17 @@ env:
 
 jobs:
   build:
+    strategy:
+      matrix:
+        java: [8, 11, 14, 17]
```

Comment:
remove matrix if you are not using it in this workflow? same comment as https://github.com/opensearch-project/observability/pull/504/files#r815045939

Reply:
I'll need to make changes to correctly use the matrix versions but we need to have it as per the release campaign